web-ng/src/app/components/project-header/project-header.component.css

```diff
@@ -14,12 +14,60 @@
  * limitations under the License.
  */
 
+.rhs-widgets {
+  float: right;
+  margin-top: 18px;
+  display: flex;
+}
+
+.logo {
+  margin-top: 16px;
+  margin-bottom: 16px;
+  margin-left: 20px;
+  vertical-align: middle;
+  width: 48px;
+  height: 48px;
+}
+
+.ground {
```

Comment:
Please use more descriptive class name, e.g. `app-title`.

Also consider specifying the font as something like "Open Sans, Arial, sans-serif"